homework/test/test-list.js

```diff
@@ -1,8 +1,36 @@
+'use strict';
 describe('Test > ', function() {
-  beforeEach(function() {
+    var toDoList, toDoItemReader;
+    beforeEach(function() {
+	toDoList = new ToDoList();
+	toDoItemReader = new ToDoItemReader();
+	toDoList._wrapper = document.createElement('div');
+	toDoItemReader._wrapper = document.createElement('div');
   });
 
   it('will test some pure functions', function() {
-    // Write any pure function assertion here.
-  });
+      toDoItemReader.start();
+      toDoList._listNoteContent[0] = {
+	  title: "This is an pen",
+	  passages: [
+	      "This is a pencil",
+	      "That is a book"
+	  ]
+      };
+      var onNoteOpenEvent = {
+	  target: {
+	      dataset: {
+		  noteId: 0,
+	      },
+	      classList: {
+		  contains: function(input){
+		      return true;
+		  }
+	      }
+	  }
+      }
```

Comment:
Why did you think you need to create a real click event? I just mentioned that the name is wrong, because in the code it listen to `click` event, but handle it with the method named as on**NoteOpen**, not on**NoteClicked**.